Server/Settings.cs

```diff
@@ -72,10 +72,19 @@ public class FlipTable {
     }
 
     public class DiscordTable {
+        public bool Enabled { get; set; } = true;
```

Comment:
```suggestion
        public bool Enabled { get; set; } = false;
```

The default should be disabled, because the `Token` and channels are missing when the `settings.json` is created from scratch.